### PR TITLE
Search Filters #Issue122 & #Issue127 & fixed design inconsistencies 

### PR DESCRIPTION
**Pull request by sabbirrifat**
You need to add this env for the G maps places autocompletion.

NEXT_PUBLIC_GOOGLE_MAPS_API_KEY

All those pages design inconsistencies were happening by tailwind are fixed in this PR

### CODE REVIEW

File: app/companies/search-talents/layout.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import React from "react";
-import SearchFilters from "./search-filters";
+// import SearchFilters from "./search-filters";
```
Comment (n1c01a5) [inline app/companies/search-talents/layout.tsx:2]: Remove dead code.

File: app/components/google-places-suggestion/google-places-suggestion.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      return Promise.resolve();
+    }
+    const src = `${GMAPS_API_URL}?key=${GMAP_API_KEY}&libraries=places&v=weekly`;
+    return loadAsyncScript(src);
+  };
+
+  const onChangeAddress = (autocomplete: any) => {
+    const address_component = autocomplete.getPlace().address_components;
+    const city = address_component.filter(
+      (f: any) =>
```
Comment (n1c01a5) [inline app/components/google-places-suggestion/google-places-suggestion.tsx:51]: Maybe google provides a type for this.

**Review by n1c01a5 — APPROVED** [review]

File: app/companies/search-talents/layout.tsx (hunk 1 above)
Comment (sabbirrifat) [reply app/companies/search-talents/layout.tsx:2]: resolved!

File: app/components/google-places-suggestion/google-places-suggestion.tsx (hunk 2 above)
Comment (sabbirrifat) [reply app/components/google-places-suggestion/google-places-suggestion.tsx:51]: not able to find any suitable types in google documentation. If I can find one then I'll replace those

**Pull request merged**